ui/raidboss/data/06-ew/trial/zodiark-ex.ts

```diff
@@ -19,6 +192,401 @@ const triggerSet: TriggerSet<Data> = {
+          const grid: boolean[] = [true, true, true, true, true, true, true, true, true];
+          for (const ex of activeExplosions) {
+            // Which y row is this explosion in?
+            let offset = -1;
+            if (ex.y < 90)
+              offset = 0;
+            if (ex.y > 90 && ex.y < 110)
+              offset = 3;
+            if (ex.y > 110)
+              offset = 6;
+            // Which x column is this explosion in?
+            if (ex.x < 90)
+              grid[offset + 0] = false;
+            if (ex.x > 90 && ex.x < 110)
+              grid[offset + 1] = false;
+            if (ex.x > 110)
+              grid[offset + 2] = false;
+          }
+          // FIXME: Commented parts prefer straight movement, loop prefers closest to northwest
+          // ++data.explosionPatternCounter;
+          // First explosion, prefer left and prefer front (melee) spot
+          // if (data.explosionPatternCounter === 1) {
+          for (let i = 0; i < grid.length; ++i) {
+            if (grid[i]) {
+              // data.lastExplosionSafe = i;
+              return output[i]!();
+            }
+          }
+          // }
+          /*
+          // 2nd or 3rd pattern. Where do we go from lastExplosionSafe?
+          const lastSafe = data.lastExplosionSafe;
+          let newSafe = -1;
+          // Can we go up?
+          if (lastSafe - 3 >= 0 && grid[lastSafe - 3])
+            newSafe = lastSafe - 3;
+          // Left?
+          else if (lastSafe - 1 >= 0 && grid[lastSafe - 1])
+            newSafe = lastSafe - 1;
+          // Right?
+          else if (lastSafe + 1 !== 4 && lastSafe + 1 !== 7 && grid[lastSafe + 1])
+            newSafe = lastSafe + 1;
+          // Down?
+          else if (lastSafe + 3 <= 8 && grid[lastSafe + 3])
+            newSafe = lastSafe + 3;
+          // Up Left?
+          else if (lastSafe - 4 >= 0 && grid[lastSafe - 4])
+            newSafe = lastSafe - 4;
+          // Up Right?
+          else if (lastSafe - 2 >= 0 && grid[lastSafe - 2])
+            newSafe = lastSafe - 2;
+          // Down Left?
+          else if (lastSafe + 2 >= 8 && grid[lastSafe + 2])
+            newSafe = lastSafe + 2;
+          // Down Right?
+          else if (lastSafe + 4 >= 8 && grid[lastSafe + 4])
+            newSafe = lastSafe + 4;
+          data.lastExplosionSafe = newSafe;
+          return output[newSafe]!();
+          */
+        }
+      },
+      outputStrings: {
+        0: Outputs.northwest,
+        1: Outputs.north,
+        2: Outputs.northeast,
+        3: Outputs.west,
+        4: Outputs.middle,
+        5: Outputs.east,
+        6: Outputs.southwest,
+        7: Outputs.south,
+        8: Outputs.southeast,
+      },
+    },
+    {
+      // 67EC is leaning left, 67ED is leaning right
+      id: 'ZodiarkEx Algedon',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: ['67EC', '67ED'], source: 'Zodiark' }),
+      alertText: (data, matches, output) => {
```

Comment:
Maybe leave a TODO if you don't want to handle this, but I think this could be an alarm after the second astral eclipse, in the case that you have to go to the opposite side: https://www.twitch.tv/videos/1244188025?t=32m20s

It may also need separate text to not conflict with explosions, e.g. "NW (knockback)"? (Maybe explosions should be info and sigils/algedon are alerts?).

Reply:
It should have the correct pacing to remind you of it after the explosions have gone off. The exact timing depends on when the explosion calls will come off should they be doable in time.